qt/scientific_interfaces/test/EnggDiffractionPresenterTest.h

```diff
@@ -124,12 +124,6 @@ class EnggDiffractionPresenterTest : public CxxTest::TestSuite {
     testing::NiceMock<MockEnggDiffractionView> mockView;
     MantidQt::CustomInterfaces::EnggDiffractionPresenter pres(&mockView);
 
-    // needs basic calibration settings from the user
```

Comment:
The calls to the tested function were unnecessary in the implementations and I removed them. It looked as if they were there to satisfy the tests without doing anything useful.